README.md

```diff
@@ -118,10 +118,15 @@ Typst's CLI is available from different sources:
   - macOS: `brew install typst`
   - Windows: `winget install --id Typst.Typst`
 
-- If you have a [Rust][rust] toolchain installed, you can also install the
-  latest development version with
-  `cargo install --git https://github.com/typst/typst --locked typst-cli`. Note that this
-  will be a "nightly" version that may be broken or not yet properly documented.
+- If you have a [Rust][rust] toolchain installed, you can also install a specific version with `--tag`. For example, you can install v0.11.0 with the command below.
+  ```
+  cargo install --git https://github.com/typst/typst --tag v0.11.0 --locked typst-cli
```

Comment:
@laurmaedj I mean, in the README this line, we could `cargo install --locked typst-cli`, which should be always equivalent with `cargo install --git https://github.com/typst/typst --tag <latest-tag> --locked typst-cli`.

Then users will install the latest released one from the registry and the maintainers will be not required to update the README when releasing.